connect_ext_ppr/models/utils.py

```diff
@@ -4,35 +4,35 @@
 
 def add_deployments(installation, listings, config, logger):
     with get_db_ctx_manager(config) as db:
-        print(db.bind)
         deployments = []
+        seen = set()
         for li in listings:
             product_id = li['product']['id']
             for hub in li['contract']['marketplace']['hubs']:
+                comb = (product_id, installation['owner']['id'], hub['hub']['id'])
                 q = db.query(Deployment).filter_by(
```

Comment:
[COD-C] I propose to check `if comb not in seen:` there to skip the query below.
Also I propose to write the following way:
```
if comb in seen:
    continue
dep = Deployment(
...
```
to reduce indentation level.
Do you agree with that?

Reply:
But `if comb not in seen` does not know if the object already exists in the DB, here we could have:

1. Combination exists in DB: for loop will continue
2. Combination does not exists in DB: `set` will be in charge of define if the combination that is going to be part of bulk create list already exists on the set of non existing DB objects.